crates/mzoon/src/wasm_opt.rs

```diff
@@ -104,16 +105,20 @@ fn unpack_wasm_opt(tar_gz: Vec<u8>) {
     for entry in archive.entries()? {
         let mut entry = entry?;
         let path = entry.path()?;
-        let file_stem = path
-            .file_stem()
+        let file_name = path
+            .file_name()
             .ok_or(anyhow!("Entry without a file name"))?;
-        if file_stem != "wasm-opt" {
-            continue;
-        }
-        let mut destination = PathBuf::from("frontend");
-        destination.push(path.file_name().unwrap());
+
+        let destination = match file_name.to_str() {
+            Some("wasm-opt")          => PathBuf::from("frontend/binaryen/bin/wasm-opt"),
+            Some("libbinaryen.dylib") => PathBuf::from("frontend/binaryen/lib/libbinaryen.dylib"),
```

Comment:
Sounds good! But this time I still use `PathBuf` since we need to dynamically construct the path for unpacked wasm-opt depending on the platform which sometimes requires file extension.